arcus_quickstart.md

```diff
@@ -0,0 +1,114 @@
+<!--
+author:   Arcus Education
+email:    arcus-support@chop.edu
+version:  1.0.0
+language: en
+narrator: US English Female
+title: Arcus Quickstart
+
+link:  https://cdn.jsdelivr.net/gh/arcus/Arcus_Labs_Orientation@main/assets/styles.css
+link:  https://cdn.jsdelivr.net/gh/arcus/education_modules@main/assets/styles.css
+script: https://kit.fontawesome.com/83b2343bd4.js
+script: https://cdn.jsdelivr.net/npm/mermaid@9.4.3/dist/mermaid.min.js
+-->
+
+# Arcus Labs: Quick Start
+
+Ready to start a research project with Arcus, or have you been given a computational lab and want to know what to do next? This quick start guide will jump start your Arcus journey! 
```

Comment:
Maybe add a description of "what is Arcus" at the very top.  In managers' meeting we joked that no one has been successful at this so honestly no pressure to get it right.  But perhaps something like "Arcus labs are a place where we can aggregate data from various sources and place that data alongside useful analysis tools.  The goal is to set you and your team up with an environment that supports reproducible data analysis projects and simplifies sharing your research insights, cohorts, data, and methods.  This can in turn help your scientific reputation and help you get the most out of your data."  IDK.